go.mod

```diff
@@ -19,14 +21,25 @@ require (
 	golang.org/x/net v0.0.0-20201224014010-6772e930b67b // indirect
 	golang.org/x/sys v0.0.0-20210113181707-4bcb84eeeb78 // indirect
 	golang.org/x/text v0.3.3 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	google.golang.org/protobuf v1.25.0 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/api v0.17.16
-	k8s.io/apimachinery v0.17.16
+	k8s.io/api v0.18.19
+	k8s.io/apimachinery v0.18.19
 	k8s.io/klog v1.0.0 // indirect
 )
 
+require (
```

Comment:
Indirekte dependencies havner i en egen blokk etter 1.17:

https://golang.org/doc/go1.17#go-command
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.